ivre/db/sql/__init__.py

```diff
@@ -1975,6 +1980,18 @@ def searchrecontype(cls, rectype):
 
     @classmethod
     def searchdns(cls, name, reverse=False, subdomains=False):
+        if isinstance(name, list):
+            if len(name) == 1:
+                name = name[0]
+            else:
```

Comment:
Just for reference. This is duplicate code with Mongo (ivre/db/mongo.py line 3020), but may be this is clearer than inheritance or subfunctions...

Reply:
I do think that it is cleaner to keep it that way, despite the (tiny) code duplication it implies.